include/alpaka/dev/cpu/SysInfo.hpp

```diff
@@ -39,223 +39,204 @@
 #include <stdexcept>
 #include <string>
 
-namespace alpaka
-{
-    namespace cpu
-    {
-        namespace detail
-        {
-            constexpr int NO_CPUID = 0;
-            constexpr int UNKNOWN_CPU = 0;
-            constexpr int UNKNOWN_COMPILER = 1;
 #if BOOST_ARCH_X86
 #    if BOOST_COMP_GNUC || BOOST_COMP_CLANG || (!BOOST_COMP_MSVC_EMULATED && defined(__INTEL_COMPILER))               \
         || BOOST_COMP_PGI
```

Comment:
```suggestion
#    if BOOST_COMP_GNUC || BOOST_COMP_CLANG || BOOST_COMP_PGI
```
We stopped supporting icpc a while ago, so you can simplify this.

Reply:
Done, thanks.